front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -73,20 +74,21 @@ const StdcmOperationalPoint = ({ point, opPointId, disabled }: StdcmOperationalP
           },
           [] as { label: string; id: string }[]
         ),
-    [point, sortedSearchResults]
+    [location, sortedSearchResults]
   );
 
   const dispatchNewPoint = (p?: SearchResultItemOperationalPoint) => {
-    if (p && 'uic' in point && p.ch === point.secondary_code && p.uic === point.uic) return;
-    const newPoint = p
+    if (p && location && p.ch === location.secondary_code && p.uic === location.uic)
```

Comment:
For all the checks like that we do, maybe we could have an util like `matchPathStepAndOp` but for stdcm ?

Reply:
Do you think this can wait the refacto of this component ?